src/index.tsx

```diff
@@ -1,20 +1,52 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import ReactDOM from 'react-dom'
-import App from './containers/App'
+import '../semantic/src/semantic.less'
 import * as config from './config.json'
 import cache from './cache'
-import '../semantic/src/semantic.less'
+import { AppWrapper } from './containers/Main'
+import { ApolloClient, ApolloProvider, NormalizedCacheObject } from '@apollo/client'
+import { persistCache } from 'apollo3-cache-persist'
+import { Loading } from './components'
+
+const App: React.FC = () => {
+  const [client, setClient] = useState<ApolloClient<NormalizedCacheObject> | undefined>(undefined)
+  useEffect(() => {
+    const client = new ApolloClient({
+      uri: config.server,
+      cache,
+    })
+
+    persistCache({
+      cache,
+      storage: window.localStorage,
+      /**
+       * Storage options.
+       */
+      // Maximum size of cache to persist (in bytes).
+      // Defaults to 1048576 (1 MB). For unlimited cache size, provide false.
+      // If exceeded, persistence will pause and app will start up cold on next launch.
+      //maxSize: false,
```

Comment:
hmm, maybe we should change this to false for now with comment to change to appropriate size. Just to avoid confusion if app cold restarts during dev

Reply:
Yes, aggreed.